feature1/one-more-file.txt

```diff
@@ -0,0 +1,4 @@
+This file was added in the feature1 branch.
```

Comment:
It's really good you have decided to add this line.

Reply:
Thanks for the comment.